recipe/meta.yaml

```diff
@@ -27,18 +29,25 @@ requirements:
     - setuptools
   run:
     - python
-    - openai
+    # openai isn't mentioned in requirements.txt
+    # but its import hardcoded as 'pip install openai==0.28.1' through import_openai()
+    # see https://github.com/zilliztech/GPTCache/blob/0.1.43/gptcache/embedding/openai.py#L8-L10
+    # because gptcache doesn't have support for openai >=1,
+    # see also https://github.com/zilliztech/GPTCache/issues/576
+    #- openai <1.0.0a1
```

Comment:
```suggestion
    #- openai <1.0.0a1
```
Thanks for adding the detailed comments! What's the purpose of this line and why is the version different from [this line](https://github.com/AnacondaRecipes/gptcache-feedstock/pull/3/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR42)?

Reply:
Fixed